tools/assetpipelineeditor/assetpipelineeditor.pro

```diff
@@ -25,7 +25,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 TEMPLATE = app
-QT += 3dcore 3drender 3dinput 3danimation 3dextras 3dquick widgets kuesa kuesa-private
+QT += 3dcore 3dcore-private 3drender 3drender-private 3dinput 3danimation 3dextras 3dquick widgets kuesa kuesa-private
```

Comment:
If I have read the code correctly, the privates are only needed if you dont have 5.13. Can you check that and only add them if that is the case?

Reply:
done